src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -509,7 +526,7 @@ public function getPermissionsInfo()
     /**
      * Returns the User's email address. If the email address is null, it is
      * assumed their eduroam address is the preferred contact method.
-     *
+     *account_locked
```

Comment:
?

Reply:
I don't know either. I removed it and added but git was having none of it

Reply:
Define "having none of it"